scss/vendor/normalize.scss

```diff
@@ -1,281 +1,184 @@
+  button {
+    -webkit-appearance: button
   }
 
-  // Remove the inner border and padding in Firefox.
-
-  button::-moz-focus-inner,
-  [type="button"]::-moz-focus-inner,
-  [type="reset"]::-moz-focus-inner,
-  [type="submit"]::-moz-focus-inner {
+  [type=button]::-moz-focus-inner,
+  [type=reset]::-moz-focus-inner,
+  [type=submit]::-moz-focus-inner,
+  button::-moz-focus-inner {
     border-style: none;
-    padding: 0;
+    padding: 0
   }
 
-  // Restore the focus styles unset by the previous rule.
-
-  button:-moz-focusring,
-  [type="button"]:-moz-focusring,
-  [type="reset"]:-moz-focusring,
-  [type="submit"]:-moz-focusring {
-    outline: 1px dotted ButtonText;
+  [type=button]:-moz-focusring,
+  [type=reset]:-moz-focusring,
+  [type=submit]:-moz-focusring,
+  button:-moz-focusring {
+    outline: 1px dotted ButtonText
   }
 
-  // Correct the padding in Firefox.
-
   fieldset {
-    padding: 0.35em 0.75em 0.625em;
+    padding: .35em .75em .625em
   }
 
-  // 1. Correct the text wrapping in Edge and IE.
-  // 2. Correct the color inheritance from `fieldset` elements in IE.
-  // 3. Remove the padding so developers are not caught out when they zero out
-  //    `fieldset` elements in all browsers.
-
   legend {
-    box-sizing: border-box; // 1
-    color: inherit; // 2
-    display: table; // 1
-    max-width: 100%; // 1
-    padding: 0; // 3
-    white-space: normal; // 1
+    box-sizing: border-box;
+    color: inherit;
+    display: table;
+    max-width: 100%;
+    padding: 0;
+    white-space: normal
   }
 
-  // Add the correct vertical alignment in Chrome, Firefox, and Opera.
-
   progress {
-    vertical-align: baseline;
+    vertical-align: baseline
   }
 
-  // Remove the default vertical scrollbar in IE 10+.
-
   textarea {
-    overflow: auto;
+    overflow: auto
   }
 
-  // 1. Add the correct box sizing in IE 10.
-  // 2. Remove the padding in IE 10.
-
-  [type="checkbox"],
-  [type="radio"] {
-    box-sizing: border-box; // 1
-    padding: 0; // 2
+  [type=checkbox],
+  [type=radio] {
+    box-sizing: border-box;
+    padding: 0
   }
 
-  // Correct the cursor style of increment and decrement buttons in Chrome.
-
-  [type="number"]::-webkit-inner-spin-button,
-  [type="number"]::-webkit-outer-spin-button {
-    height: auto;
+  [type=number]::-webkit-inner-spin-button,
+  [type=number]::-webkit-outer-spin-button {
+    height: auto
   }
 
-  // 1. Correct the odd appearance in Chrome and Safari.
-  // 2. Correct the outline style in Safari.
-
-  [type="search"] {
-    -webkit-appearance: textfield; // 1
-    outline-offset: -2px; // 2
+  [type=search] {
+    -webkit-appearance: textfield;
+    outline-offset: -2px
   }
 
-  // Remove the inner padding in Chrome and Safari on macOS.
-
-  [type="search"]::-webkit-search-decoration {
-    -webkit-appearance: none;
+  [type=search]::-webkit-search-decoration {
+    -webkit-appearance: none
   }
 
-  // 1. Correct the inability to style clickable types in iOS and Safari.
```

Comment:
Normally there should be still the comments.

Please use the original file, not the production file.

https://github.com/necolas/normalize.css/blob/8.0.1/normalize.css#L304